src/App.js

```diff
@@ -1,16 +1,44 @@
 import React from 'react';
 import './App.scss';
 
+import { Counter } from './components/Counter';
+import 'semantic-ui-css/semantic.min.css';
+
 class App extends React.Component {
-  addOne = () => {};
+  state = {
+    count: 0,
+  }
+
+  addOne = () => {
+    this.setState(state => ({
+      count: state.count + 1,
+    }));
+  };
+
+  add100 = () => {
+    this.setState(state => ({
+      count: state.count + 100,
+    }));
+  };
 
-  add100 = () => {};
+  increase = () => {
+    const { count } = this.state;
 
-  increase = () => {};
+    (count % 5 === 0)
+      ? this.add100(this.addOne())
```

Comment:
такие конструкции лучше избегать
```suggestion
      ? this.add100()
```